app/helpers/patients_helper.rb

```diff
@@ -1,4 +1,7 @@
 # frozen_string_literal: true
 
 module PatientsHelper
+  def display(attribute)
+    attribute
```

Comment:
What is up with this method?

Reply:
I originally had it spitting out an html fragment. It was just a few tags, so a partial would have been more text than just copy/pasting it, so I decided to create a helper method. Rubocop didn't like me using raw() in the helper so I got rid of it and the html and just forgot to remove display(). I'll remove it since you're redesigning the UI anyways.